indico/htdocs/js/indico/modules/sessions.js

```diff
@@ -0,0 +1,130 @@
+(function(global) {
+    'use strict';
+
+    function handleRowSelecteion() {
+        $('table.i-table input.select-row').on('change', function() {
+            $(this).closest('tr').toggleClass('selected', this.checked);
+            $('.js-requires-selected-row').toggleClass('disabled', !$('.sessions input:checkbox:checked').length)
+        }).trigger('change');
+    }
+
+    function setupTableSorter() {
+        $('.sessions .tablesorter').tablesorter({
+            cssAsc: 'header-sort-asc',
+            cssDesc: 'header-sort-desc',
+            cssInfoBlock: 'avoid-sort',
+            cssChildRow: 'session-blocks-row',
+            headerTemplate: '',
+            sortList: [[1, 0]]
+        });
+    }
+
+    global.setupSessionsList = function setupSessionsList() {
+        handleRowSelecteion();
+        setupTableSorter();
+
+        $('#select-all').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', true).trigger('change');
+        });
+
+        $('#select-none').on('click', function() {
+            $('table.i-table input.select-row').prop('checked', false).trigger('change');
+        });
+
+        $('.sessions .show-session-slots').on('click', function() {
+            $(this).parent().parent().next('tbody.session-blocks').toggle();
```

Comment:
that's VERY specific to a specific DOM structure. why not give the parent's parent a class and use `.closest()`?